scripts/macos-gen-dmg.py

```diff
@@ -11,6 +11,34 @@
 import os
 from sys import platform
 
+def package_dmg(install_path, destination):
+    # for compiling on Darwin as 'genisoimage' doesn't exist
+    # uses 'create-dmg' (brew install create-dmg)
+    if platform == "darwin":
+        command = shlex.split('''create-dmg
+            --volname "Taisei"
+            --volicon "Taisei.app/Contents/Resources/Taisei.icns"
+            --window-pos 200 120
+            --window-size 550 480
+            --icon-size 64
+            --icon "Taisei.app" 100 50
+            --icon "README.txt" 50 200
+            --icon "STORY.txt" 200 200
+            --icon "GAME.html" 350 200
+            --icon "COPYING" 125 350
+            --icon "ENVIRON.html" 275 350
+            --hide-extension "Taisei.app"
+            --app-drop-link 300 50
+            {0} {1}'''.format(destination, str(install_path)))
```

Comment:
Splicing paths into the string is a bad idea; `shlex.split` is going to mishandle paths that contain whitespace or other significant characters like `'` and `"`. They should be appended to `command` (a list) directly instead, as they were before. In the `else` clause it is done correctly as well.